https://git.savannah.gnu.org/cgit/rcs.git/commit/?h=p&id=f2330a6268d244e97f47a97f4767736fc3b31455
https://bugs.gentoo.org/820230

From: Thien-Thi Nguyen <ttn@gnu.org>
Date: Thu, 5 Apr 2018 16:20:35 +0200
Subject: [v] Avoid "test FILE1 -nt FILE2".

* tests/t810: Use "btdt mtimecmp" instead of "test -nt".
Thanks to both sergio and Xose Vazquez Perez
for indirectly signalling this problem.  See 2018-04-05,
"Add support for "btdt mtimecmp FILE1 FILE2"" for details.
--- a/tests/t810
+++ b/tests/t810
@@ -27,7 +27,8 @@ split_std_out_err no
 
 must 'echo new line >> $w'
 must 'ci -mm -l -d -T $w'
-test $w -nt $v && problem "$w newer than $v"
+test 1 = `./btdt mtimecmp $w $v` \
+    && problem "$w newer than $v"
 
 exit 0
 
cgit v1.2.1
